opal/mca/btl/portals4/configure.m4

```diff
@@ -11,7 +11,7 @@
 # Copyright (c) 2004-2005 The Regents of the University of California.
```

Comment:
This should not be here ? Did you missed a merge or maybe you rebase according to OMPI master and the ICLDisco master is slightly behind ?